src/cloud/lib/charts/charts.tsx

```diff
@@ -37,3 +41,69 @@ export const Chart = ({ config, isYml = false }: ChartProps) => {
     </div>
   )
 }
+
+interface RehypeChartProps {
+  tagName: string
+  isYml?: boolean
+}
+
+export function createAndAddElementWithId(id: string, tagName: string) {
+  let chartExportElement = window.document.getElementById(id)
+  if (chartExportElement == null) {
+    chartExportElement = window.document.createElement(tagName)
+    chartExportElement.id = id
+    chartExportElement.style.display = 'none'
+    const bodyElement = window.document.getElementsByTagName('body').item(0)
+    if (bodyElement) {
```

Comment:
@Rokt33r We could add it to some other element if this is not ok, but it needs to be added to DOM to be able to work for export.

Reply:
will refactor